x/ibc/02-client/keeper/client.go

```diff
@@ -140,7 +140,7 @@ func (k Keeper) CheckMisbehaviourAndUpdateState(ctx sdk.Context, misbehaviour ex
 	switch e := misbehaviour.(type) {
 	case ibctmtypes.Evidence:
 		clientState, err = tendermint.CheckMisbehaviourAndUpdateState(
-			clientState, consensusState, misbehaviour, consensusState.GetHeight(), ctx.BlockTime(),
+			clientState, consensusState, misbehaviour, consensusState.GetHeight(), ctx.BlockTime(), ctx.ConsensusParams(),
```

Comment:
Why not just pass the Context?

Reply:
bc that would involve a major refactor of the tests and suite. I'll add it to the list of items in #5558 👍

Reply:
related to #6064 as well